docs/azure-blob-storage.md

```diff
@@ -0,0 +1,40 @@
+# Azure Storage Utilities for Data Operations
+
+## Configure Environment Variables
+
+Before you begin utilising the scripts, you must configure the necessary environment variables. These variables will be used to access Azure Blob Storage.
+
+### Setting Up Environment Variables on Linux and macOS
+
+Open your terminal.
+Open your `.bashrc` file in a text editor. You can use `nano` by typing
+
+```bash
+nano ~/.bashrc
+```
+
+Add the following lines at the end of the .bashrc file to set your environment variables
+
+```bash
+   export STORAGE_SAS_TOKEN_CHD="sas_token_placeholder"
+   export CONTAINER_NAME_CHD="container_name_placeholder"
+   export STORAGE_ACCOUNT_CHD="storage_account_placeholder"
+   ```
+
+   Replace sas_token_placeholder, container_name_placeholder, and storage_account_placeholder with your actual Azure Blob Storage details.
+
+Save and close the editor. If you're using nano, you can do this by pressing CTRL+X, then Y to confirm, and Enter to save.
+
+Update your current session with the changes by sourcing the .bashrc file
+
+```bash
+   source ~/.bashrc
+   ```
+
+### Install Required Packages
```

Comment:
Please remove this installation instruction, because it is misleading.

You have already added the `azure-storage-blob` in pyproject.toml and lock file by running the command and you have correctly pushed the changes to the GitHub. All people need to do is to run `make clean .venv` to reinstall their poetry dependencies after pulling the latest update.

Installing dependencies is covered in mair README, so you are fine here